adjust-downstream-webhook/README.md

```diff
@@ -12,7 +12,7 @@ Since each version of the webhook is one-to-one compatible with a specific versi
 ```bash
 ## Create a token through the UI. The token should have no scope and be made for a user who is a global admin.
 read -s RANCHER_TOKEN && export RANCHER_TOKEN
-## The server URL for Rancher - you can get this value in the "server-url" setting.
+## The server URL for Rancher - you can get this value in the "server-url" setting. The example format should be: https://rancher-test.home
```

Comment:
Maybe mention that you can find this in in Rancher global settings > settings > server-url.

Reply:
@oleg-vorobiov-suse would you be able to add this? If you agree, looks like we can merge once added

Reply:
Just added Jeff's changes, I am good